### PR TITLE
add previous

### PR DESCRIPTION
**Pull request by lvjing2**
Support the get the latest previous activation time.

### CODE REVIEW

**Comment by lvjing2** [comment]
@robfig Hi, How do you think if add the previous in this MR?

**Comment by max-wittig** [comment]
Thanks for this MR. I would also appreciate this feature. See: #224

File: cron.go [hunk 1]
```diff
@@ -36,6 +36,9 @@ type Schedule interface {
 	// Next returns the next activation time, later than the given time.
 	// Next is invoked initially, and then each time the job is run.
 	Next(time.Time) time.Time
+	// Previous returns the next activation time, smaller or equal with the given time.
+	// Previous is invoked initially, and then each time the job is run.
+	Previous(time.Time) time.Time
```
Comment (lonre) [inline cron.go]: per https://github.com/robfig/cron/blob/e843a09e5b2db454d77aad25b1660173445fb2fc/cron.go#L53-L59

maybe it is better to rename this from `Previous` to `Prev`

Comment (lvjing2) [reply cron.go]: done

**Comment by max-wittig** [comment]
Hi @robfig. Seems like it was renamed by the creator. Care to review again? Would love to have this!

**Comment by lvjing2** [comment]
@robfig Hi, Is there some problem in this MR? If yes, would you like to share it to us?

**Comment by robfig** [comment]
I'm not sure.. This seems like a lot of code for little benefit. Schedule is an interface, so folks can customize it for their use case. You can have a PrevSchedule that implements this if you wish. I don't want to support this use case, though. Sorry!

**Comment by vllry** [comment]
Thanks @lvjing2 for the example implementation. This is extremely useful when identifying a backlog of missed runs.

**Pull request closed without merging**